src/components/shared/HamburgerMenu.less

```diff
@@ -0,0 +1,24 @@
+.web-logo {
+  display: flex;
+  justify-content: center;
+  padding: 20px 0px;
+  img {
+    width: 180px;
+  }
+}
+
+.ant-drawer .ant-drawer-body {
```

Comment:
It's not good to override AntDesign class names globally, this may lead to unexpected style changes in other places where those components are used. Let's always wrap it with some custom class. For example in this case let's add something like `side-menu-drawer` class to `Drawer` and wrap all these styles with it. Also please use nested styles to avoid duplication of class names (like `.ant-drawer` f.e)